scripts/db-migration.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
This script provides a way to automatically update the migration auto-generated file so that it is converted to a seed migration.  Which means it can run scripts in the appropriate folder structure.

Reply:
This is nice.